plugins/modules/katello_repository_set.py

```diff
@@ -152,7 +152,6 @@
 def get_desired_repos(desired_substitutions, available_repos):
```

Comment:
Should this now be `return itertools.chain(filter(...) for sub in desired_substitutions)`?

Reply:
maybe? ;)

Reply:
`TypeError: 'filter' object is not subscriptable` and I am not in the mood to understand why ;)